config/runtimes/vllm-gaudi-template.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  labels:
+    opendatahub.io/dashboard: 'true'
+    opendatahub.io/ootb: 'true'
+  annotations:
+    description: vLLM ServingRuntime to support Gaudi(for Habana AI processors)
+    openshift.io/display-name: vLLM Gaudi ServingRuntime for KServe
+    openshift.io/provider-display-name: Red Hat, Inc.
+    tags: rhods,rhoai,kserve,servingruntime
+    template.openshift.io/documentation-url: https://github.com/opendatahub-io/vllm
+    template.openshift.io/long-description: This template defines resources needed to deploy vLLM ServingRuntime with KServe in Red Hat OpenShift AI
+    opendatahub.io/modelServingSupport: '["single"]'
+    opendatahub.io/apiProtocol: 'REST'
+  name: vllm-gaudi-runtime-template
+objects:
+  - apiVersion: serving.kserve.io/v1alpha1
+    kind: ServingRuntime
+    metadata:
+      name: vllm-gaudi-runtime
+      annotations:
+        openshift.io/display-name: vLLM Gaudi ServingRuntime for KServe
+        opendatahub.io/recommended-accelerators: '["habana.ai/gaudi"]'
+      labels:
+        opendatahub.io/dashboard: 'true'
+    spec:
+      annotations:
+        prometheus.io/port: '8080'
+        prometheus.io/path: '/metrics'
+      multiModel: false
+      supportedModelFormats:
+        - autoSelect: true
```

Comment:
This SR would be the 3rd runtime with auto select enabled.

I know the dashboard doesn't use auto selection. But, perhaps, can we disable auto-select from the less used vLLM runtimes? This is just to ensure that for users not using the dashboard the most common runtime would be selected (thinking in modelcars/rawdeployment features which won't have UI for now).